artemis/src/main/java/tech/pegasys/artemis/datastructures/beaconchainblocks/BeaconBlock.java

```diff
@@ -29,17 +29,9 @@
   // Body
   private BeaconBlockBody body;
 
+  public BeaconBlock() {}
 
-  public BeaconBlock() {
-
-  }
-
-  /*********************
-   *                   *
-   * GETTERS & SETTERS *
```

Comment:
Can we have the getter & setter comments stay in?

Reply:
NVM - I see that the format was just changed

Reply:
Oh, darn. I guess the google formatter doesn't like flower boxes. I'll add an issue to evaluate an override to keep those if we prefer, or if nothing else at least clean up the number of asterisks after the auto-reformat.